### DISCUSSION TITLE
?? Presidence

### DISCUSSION THREAD

**Discussion by TonyValenti** [dotnet/csharplang#2625] [Language Ideas] [answered]
Hi All,
I often find myself writing code similar to the following:
````
if ((SourceDataModel?.TextSnippets?.Count ?? 0) == 0) {               
    ....
}
````

Is there a way for me to rewrite my IF statement so that I don't need the extra parenthesis?

**Comment by yaakov-h** [comment, marked as answer]
Potentially with #1350, `SourceDataModel?.TextSnippets?.Count is 0 or null`?

**Comment by svick** [comment]
You could write it as:

```c#
if (!SourceDataModel?.TextSnippets?.Any() ?? true) {
    ....
}
```
